src/main/resources/db/changelog/changes/v1.1.0/2022-06-01-MODINREACH-287.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
v1.1.0 has been released already
you shouldn't place files in "..\v1.1.0" directory anymore